scripts/moneat-agent/moneat_agent/reviews.py

```
@@ -0,0 +1,132 @@
+class ReviewSnapshot:
+    comments: list[ReviewComment] = field(default_factory=list)
+
+    @property
+    def ids(self) -> set[int]:
+        return {c.id for c in self.comments}
+
+
+def fetch_review_comments(repo: str, pr_number: int) -> ReviewSnapshot:
+    """Fetch all review comments on a PR."""
+    try:
+        raw = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)